templateutils/example_test.go

```diff
@@ -50,7 +50,6 @@ func ExampleGenerateUsageUndecorated() {
 	i := struct {
 		X       int
 		Y       string
-		hidden  float64
```

Comment:
Yeah, and I think this is overzealous with fields that are actually read by code that is excluded (i.e. in vendor) and is breaking BAT. So i'll refine those removed by "unused" and also not add it to the travis builds.